dev/wlp-jakartaee-transform/rules/jakarta-bundles.properties

```diff
@@ -1 +1,11 @@
-*=*.jakarta,2.0,+" Jakarta",+"; Jarkata Enabled"
\ No newline at end of file
+*=*.jakarta,2.0,+" Jakarta",+"; Jarkata Enabled"
+
+com.ibm.ws.security.authentication.tai=io.openliberty.security.authentication.internal.tai,,"Security TAI Authentication","Security TAI Authentication; Jarkata Enabled"
```

Comment:
Jarkata should be Jakarta.  Same on line 1.

Reply:
potato, potatoe... jakarta, jarkarta... fixed.

Notice that the first line (that I did not modify beyond the new line) has "Jarkarta". I fixed that too. That's what I get for copy / paste.